src/main/resources/templates/help.html

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
-<html xml:th="http://www.tymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
```

Comment:
XML Namespace for ThymeLeaf must be defined using `xmlns:th` not `xml:th`. [Reference](https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XUL/Namespaces)

`<html>` tag must have lang attribute defined. [Reference](https://www.w3.org/International/questions/qa-html-language-declarations)